tests/framework/results/test_context.py

```diff
@@ -420,34 +419,6 @@ def test_gather_results_with_no_stratifications():
     )
 
 
-def test__bad_aggregator_return():
```

Comment:
We could still have this test with an aggregator returning something that isn't a DataFrame or a Series.

Reply:
But doesn't .apply on a df always return one of those two?

Reply:
In fact, I think we can double down and not only remove the test but actually remove the isinstance check